### ISSUE TITLE
Understanding the Role of Individual Units in a Deep Neural Network

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#119]
# TL;DR
Research to interpret the role of each unit in DNNs. They found that there are units that learn concepts such as "tree" without explicitly giving them to the DNN. In GAN, They have succeeded in reducing trees from the image and attaching doors to buildings by manipulating the units that control each concept.
![image](https://user-images.githubusercontent.com/24524018/93746290-d4148b00-fc2f-11ea-9732-1e3c8f6ab12d.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.05041

# Submission Dates(yyyy/mm/dd)
2020/09/12

# Authors and institutions
David Bau, Jun-Yan Zhu, Hendrik Strobelt, Agata Lapedriza, Bolei Zhou, Antonio Torralba

# Methods

# Results

# Comments
